tests/configuration/permissions_test.js

```diff
@@ -257,3 +257,15 @@ Scenario(
     I.waitForText('Insufficient access permissions.', 10, ruleTemplatesPage.elements.unathorizedMessage);
   },
 );
+
+Scenario(
+  'PMM-T2009 - Verify that editor user can see failed advisors data on home dashboard @nightly @grafana-pr',
+  async ({ I, dashboardPage }) => {
+    await I.Authorize(users.editor.username, users.editor.password);
+    I.amOnPage(dashboardPage.homeDashboard.url);
+    await I.waitForVisible(dashboardPage.homeDashboard.panels.failedAdvisorsValue());
+    const failedAdvisorsCount = await I.grabTextFrom(dashboardPage.homeDashboard.panels.failedAdvisorsValue());
+
+    I.assertEqual(failedAdvisorsCount, '0', 'Failed Advisors count should be visible and should be 0');
```

Comment:
And improvement later would be to check actual counts and not zero (since it's a default value, and if the failed checks wouldn't be fetched for whatever reason, it could lead into a false positive)